dpcpp/matrix/sparsity_csr_kernels.dp.cpp

```diff
@@ -33,28 +33,14 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "core/matrix/sparsity_csr_kernels.hpp"
 
 
-#include <algorithm>
-#include <numeric>
-#include <utility>
-
-
-#include <CL/sycl.hpp>
-
-
 #include <ginkgo/core/base/exception_helpers.hpp>
-#include <ginkgo/core/base/math.hpp>
-#include <ginkgo/core/matrix/dense.hpp>
-
-
-#include "core/base/iterator_factory.hpp"
-#include "dpcpp/components/format_conversion.dp.hpp"
 
 
 namespace gko {
 namespace kernels {
 namespace dpcpp {
 /**
- * @brief The SparsityCsr pattern format namespace.
+ * @brief The Compressed sparse row matrix format namespace.
```

Comment:
That doesn't seem entirely correct, since it is only the pattern, not the values.

Reply:
this gives the same name as csr?

Reply:
Isn't it confusing between csr and SparsityCsr? they give the same brief